exercises/rh/ex1/e1.f90

```diff
@@ -1,25 +1,105 @@
 program e1
-
+  ! Import the modules that we are going to use
   use geometry
   use particle
   implicit none
 
-INTEGER :: i
-INTEGER :: n ! number of particles
-type(particle3d), dimension(:), allocatable :: particles ! particle type array (contains the particles info)
-real, dimension(:), allocatable :: m
+  INTEGER :: i, j ! loop indexes
+  INTEGER :: rc ! variable to read line of input file (with the i.c.)
+  INTEGER :: n = 0 ! number of particles
+  REAL :: dt, t_end, t, dt_out, t_out = real64 ! time variables
```

Comment:
Great work, but you might still want to take a look at how to set the kind of your real variables from the [lecture slides](https://github.com/hpparvi/ull_tp_2425/blob/main/lectures/l03/l03b.pdf).